generators/server/templates/src/test/java/package/web/rest/_AccountResourceIntTest.java

```diff
@@ -243,20 +288,24 @@ public void testRegisterEmailEmpty() throws Exception {
     @Transactional<% } %>
     public void testRegisterDuplicateLogin() throws Exception {
         // Good
-        UserDTO u = new UserDTO(
+        ManagedUserDTO u = new ManagedUserDTO(
+            null,                   // id
             "alice",                // login
             "password",             // password
             "Alice",                // firstName
             "Something",            // lastName
             "alice@example.com",    // e-mail
             true,                   // activated
-            "<%= nativeLanguageShortName %>",                   // langKey
-            new HashSet<>(Arrays.asList(AuthoritiesConstants.USER))
-        );
+            "<%= nativeLanguageShortName %>",               // langKey
+            new HashSet<>(Arrays.asList(AuthoritiesConstants.USER))<% if (databaseType == 'mongodb' || databaseType == 'sql') { %>,
+            null,                   // createdDate
+            null,                   // lastModifiedBy
+            null                    // lastModifiedDate 
+        <% } %>);
 
         // Duplicate login, different e-mail
-        UserDTO dup = new UserDTO(u.getLogin(), u.getPassword(), u.getLogin(), u.getLastName(),
-            "alicejr@example.com", true, u.getLangKey(), u.getAuthorities());
+        ManagedUserDTO dup = new ManagedUserDTO(u.getId(), u.getLogin(), u.getPassword(), u.getLogin(), u.getLastName(),
```

Comment:
Same here, dup means nothing, could you please rename this variable to duplicatedUser or duplicate?